templates/web/common/includes/list_of_products.tt.html

```diff
@@ -2,11 +2,11 @@
 <!-- start templates/[% template.name %] -->
 
 <p>[% error %]</p>
-[% IF (current_link_query.defined) && !(jqm.defined) %]
+[% IF (current_link.defined) && !(jqm.defined) %]
     [% IF country != 'en:world' %]
-        <p>&rarr; <a href= "[% world_subdomain %][% current_link_query %]"> [% lang('view_results_from_the_entire_world') %]</a></p>
+        <p>&rarr; <a href= "[% world_subdomain %][% current_link %]"> [% lang('view_results_from_the_entire_world') %]</a></p>
```

Comment:
I'm not sure if in that kind of cases the existence of $current_link_query, was not a marker that we were on a search results page (but maybe this template is only used in the context of a search result, in which case this is ok)

Reply:
It might be a false alarm, as it seems list_of_products.tt.html is used only for search results.

Reply:
The template is used whenever we display a list of products (e.g. the front page, facets, search results etc.).
e.g. https://uk.openfoodfacts.org/label/organic

Currently we do not display the link to "world" when we are on the home page of a country.

Reply:
http://world.openfoodfacts.localhost/label/organic seems to work on this patch. Though my testing is entirely based off of looking at the dev env after applying the patch. It is surely not a full test, and as mentioned here and above, I'm missing links. to look at, and am likely to be missing more.

How should this PR proceed, does it need changes, or it is too unclear on if it could introduce a problem and should be closed?